gdal/data/FGD_GMLSchema.xsd

```diff
@@ -0,0 +1,842 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xs:schema targetNamespace="http://fgd.gsi.go.jp/spec/2008/FGD_GMLSchema"
```

Comment:
Is this schema needed at all in gdal/data ?
Has it a open-source compatible license ?